src/components/Menu/MenuItem.tsx

```diff
@@ -59,6 +60,9 @@ class MenuItem extends UIComponent<Extendable<IMenuItemProps>, any> {
     /** Shorthand for primary content. */
     content: PropTypes.any,
 
+    /** A menu item can show it is currently unable to be interacted with. */
+    disabled: PropTypes.bool,
```

Comment:
We are missing example for `disabled`.

Reply:
https://github.com/stardust-ui/react/issues/186 Added a feature request. Thanks!